site2/docs/functions-overview.md

```diff
@@ -5,30 +5,26 @@ sidebar_label: "Overview"
 ---
 
 This section introduces the following content:
-* [What is Pulsar Functions](#what-is-pulsar-functions)
+* [What are Pulsar Functions](#what-is-pulsar-functions)
 * [Why use Puslar Functions](#why-use-pulsar-functions)
 * [Use cases](#use-cases)
 * [User flow](#user-flow)
 
 
 ## What are Pulsar Functions
+Pulsar functions are a serverless computing framework that runs on top of Pulsar. You can use this simple API and execution framework to implement common use cases, such as filtering and routing. Once you write a Pulsar function and submit it to the Pulsar cluster, it is up and running immediately.
 
-Pulsar Functions are a serverless computing framework that runs on top of Pulsar and processes messages in the following way:
-* consumes messages from one or more topics,
-* applies a user-defined processing logic to the messages,
-* publishes the outputs of the messages to other topics.
-
-The following figure illustrates the computing process of a function. 
+The diagram below illustrates the three steps in the functions computing process. 
 
 ![Pulsar Functions execute user-defined code on data published to Pulsar topics](/assets/function-overview.svg)
 
-A function receives messages from one or more **input topics**. Each time messages are received, the function completes the following steps:
-1. Consumes the messages in the input topics.
-2. Applies a customized processing logic to the messages and:
-    a) writes output messages to an **output topic** in Pulsar
-    b) writes logs to a **log topic** if it is configured (for debugging purposes)
-    c) writes [state](functions-develop-state) to BookKeeper (if it is configured) 
-
+Each time a function receives a message, it completes the following consume-apply-publish steps.
+1. Consumes the message from one or more **input topics**. 
+2. Applies the customized (user-supplied) processing logic to the message.
+3. Publishes the output of the message to other topics and writes:
+    a) output messages to an **output topic** in Pulsar
+    b) logs to a **log topic** (if it is configured)for debugging
+    c) [state](functions-develop-state) updates to BookKeeper (if it is configured) 
```

Comment:
Thanks for your contribution!

1. Before submitting a doc PR, please [preview your changes locally](https://docs.google.com/document/d/1wszdtMRo6MhKbVaggPK7_bnKaC4TewuT--GWZZxJNGg/edit#). When submitting a PR, please attach the screenshots of your local preview. In this way, you can get your PR reviewed and merged more quickly.

2. Does this change apply to multiple doc versions? If so, please [update them all](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs).

Thank you! 😊